### PR TITLE
Position activated in the PositioningSettings and improvement

### PR DESCRIPTION
**Pull request by signedav**
Storing the settings with the Qt Setting object.

The PositioningSettings object is excluded into its own file.
PositioningSettingsPopup and the settings of the position (activated) is stored there.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-f531b2f7a0e328aaa94a2c93b624dc53af84c9f6-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-f531b2f7a0e328aaa94a2c93b624dc53af84c9f6-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-f531b2f7a0e328aaa94a2c93b624dc53af84c9f6-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-f531b2f7a0e328aaa94a2c93b624dc53af84c9f6-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-cdf6c3f9c89643eaa72acdb9c37c29e133ea42f7-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-cdf6c3f9c89643eaa72acdb9c37c29e133ea42f7-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-cdf6c3f9c89643eaa72acdb9c37c29e133ea42f7-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-cdf6c3f9c89643eaa72acdb9c37c29e133ea42f7-x86_64.apk)

File: src/qml/PositioningSettings.qml [hunk 1]
```diff
@@ -1,134 +1,12 @@
...
-          text: qsTr( "Use the altitude as reported by the positioning interface. Skip any altitude correction that may be implied by the coordinate system transformation." )
-          font: Theme.tipFont
-
-          wrapMode: Text.WordWrap
-          Layout.fillWidth: true
-        }
-      }
-    }
-  }
+  property string antennaHeight: "0"
```
Comment (m-kuhn) [inline src/qml/PositioningSettings.qml]:
```suggestion
  property real antennaHeight: 0.0
```

What do you think?

Comment (signedav) [reply src/qml/PositioningSettings.qml]: I did it. Check my last commit.

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-3fa456057613c2b2a690a7c97e6bae0409f431f2-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-3fa456057613c2b2a690a7c97e6bae0409f431f2-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-3fa456057613c2b2a690a7c97e6bae0409f431f2-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-882-3fa456057613c2b2a690a7c97e6bae0409f431f2-x86_64.apk)

**Pull request merged**